flexget/plugins/input/medusa.py

```diff
@@ -0,0 +1,125 @@
+from __future__ import unicode_literals, division, absolute_import
+from builtins import *  # noqa pylint: disable=unused-import, redefined-builtin
+from future.moves.urllib.parse import urlparse
+
+import logging
+
+from flexget import plugin
+from flexget.event import event
+from flexget.entry import Entry
+
+log = logging.getLogger('medusa')
+
+
+class Medusa(object):
+    schema = {
+        'type': 'object',
+        'properties': {
+            'base_url': {'type': 'string', 'format': 'uri'},
+            'port': {'type': 'number', 'default': 8081},
+            'username': {'type': 'string'},
+            'password': {'type': 'string'},
+            'only_monitored': {'type': 'boolean', 'default': False},
+            'include_ended': {'type': 'boolean', 'default': False},
+        },
+        'required': ['username', 'password', 'base_url'],
+        'additionalProperties': False
+    }
+
+    headers = {
```

Comment:
this is redundant, as it is implicitly added when you use the `json` keyword when sending requests